intro-to-R-tidyverse/02-intro_to_ggplot2.Rmd

```diff
@@ -32,7 +32,8 @@ We performed three sets of contrasts:
 **More ggplot2 resources:**  
 
 - [ggplot2 website](https://ggplot2.tidyverse.org/)  
-- [Handy cheatsheet for ggplot2 (pdf)](https://github.com/rstudio/cheatsheets/raw/main/data-visualization.pdf)  
+- [Handy cheatsheet for ggplot2 (pdf)](https://github.com/rstudio/cheatsheets/raw/main/data-visualization.pdf)
+- [_Data Visualization, A practical introduction_](https://socviz.co/)
```

Comment:
Added this resource, the free draft of which is online. Feel free to remove this if overkill.